src/engine/renderer/glsl_source/reliefMapping_fp.glsl

```diff
@@ -201,10 +201,8 @@ void computeDLights( vec3 P, vec3 N, vec3 I, vec4 diffuse, vec4 specular,
 
 vec3 ComputeNormalInTangentSpace(sampler2D normalMap, vec2 texNormal)
 {
-	vec3 N = texture2D(u_NormalMap, texNormal.st).xyw;
-	N.x *= N.z;
-	N.xy = 2.0 * N.xy - 1.0;
-	N.z = sqrt(1.0 - dot(N.xy, N.xy));
+	vec3 N = texture2D(normalMap, texNormal.st).rgb;
```

Comment:
I guess this commit should be removed now that Fuma has explained why things are extra magical?

Reply:
more code is required to get it working properly in every case, this line:
```
vec3 N = texture2D(u_NormalMap, texNormal.st).xyw; 
```

means that normalmap is sampled from rga for each case, even non DXn normal maps with heightmap in alpha channel. Because of heightmap in alpha channel use case we may use a “_if DXn_” switch (the code only claims to not require such test when the only faced use cases are RGB files or DXn ones abusing Alpha.

Reply:
So why is it a good idea to remove support of all the formats other than the "uncompressed RGB" one? Did you find that no assets use any of them?

Reply:
It's not a good idea at all. All Unvanquished maps use the other format.

I fixed it.